src/main/java/analyzer/exercises/loglevels/UseSubstringMethod.java

```diff
@@ -2,16 +2,30 @@
 
 import analyzer.Comment;
 
+import java.util.Map;
+
 /**
  * @see <a href="https://github.com/exercism/website-copy/blob/main/analyzer-comments/java/log-levels/use_substring_method.md">Markdown Template</a>
  */
 class UseSubstringMethod extends Comment {
+    private final String onMethod;
```

Comment:
_Very_ small improvement: you don't use substring _on_ a method, but _in_ a method. So I suggest renaming the parameter to `inMethod`.

Reply:
Done!